CONTRIBUTING.md

```diff
@@ -26,26 +26,156 @@
+
+#### Triage meetings
+Triage meetings are held on a weekly basis, with ad hoc meetings held as required. Their main purpose is to review what has been achieved since the last meeting and provide an opportunity to discuss specific cards if there is concern about their current priority. The Waffle board is used to provide this information.
+
+##### Triage meeting tasks
+* Review tasks completed since the last meeting
+* Review new issues in the backlog
+* Allow the opportunity to discuss the classification of any open issue that has not already been discussed in the meeting, this should not be used as a chance to rehash old arguments but instead to highlight the case for issues where the circumstances have changed.
+
+## Backlog -> Approved
+Approved is updated with tasks from Backlog if Approved is under its WIP limit. In general the next card to move to Approved will be decided based on priority and age, with the oldest card of the highest priority the next moved. This task will generally be performed by the PTL or Bernardo as and when Approved moves below its WIP limit.
+
+## Approved
+This column shows the work to which the engineering team has currently committed but not yet started.
+
+## Approved -> Doing
+### Selecting an issue to work on
+Working on a task means moving it from Approved to Doing, this operation is only allowed if it doesn't violate the Doing WIP limit. All available work is found on the Waffle board in Approved, the next task is always at the top. Assign yourself as the owner of the issue at the top and move it from the top of Approved to the bottom of Doing.
+
+If a task includes an existing pull request from someone outside the engineering team, they should not be assigned as the owner. All work must be owned by the engineering team, if someone on another team e.g. support has provided code then work with them to get it merged but ultimately it is the responsibility of the engineering team to manage the flow of work and ensure all aspects of a card are completed.
+
+## Doing
+#### Fixing an issue
+The issue description contains a list of the steps required to close an issue, this is added by the [issue template](.github/ISSUE_TEMPLATE.md). As each step is completed, update the description.
+
+##### New issues discovered while working an issue
+If a new bug or feature requirement is found while working on an assigned issue, that new work item should be logged as a separate issue in Backlog.
+
+If the new issue blocks the existing issue, you should assign yourself to the new issue and move it to Doing. The existing issue should be updated with all the work that has been done, the label 'status-blocked' added and a link to the blocking issue should be included in the issue description. In this situation the WIP limit can be exceeded. Similar to situations where the WIP limit prevents new work being started the team needs to work to reduce the WIP so that new tasks can be started.
```

Comment:
@b3rnard0 My thinking was that the WIP limits should guide this. So if the column is at the WIP limit, this prevents individuals starting new tasks and so they should look to help with existing tasks. If no WIP limit is reached, does it matter? In another comment you mention we need to be aware of cycle time and so in that case maybe that would be another reason to assist on the blockage. By default though I wonder if it is necessary - I may just need to read more kanban.